### DISCUSSION TITLE
Issue categorisation

### DISCUSSION THREAD

**Discussion by Tamschi** [snarfed/bridgy-fed#1407] [Ideas]
I'd like to organise the issues a bit more, to make them easier to browse and hopefully prioritise.

I was thinking about introducing the following labels:

| name | colour | description | example
|-|-|-|-|
| `feature` | a blue | Features and feature requests that are specific to Bridgy Fed, not fully described by the protocols. | most DM-related functions |
| `parity` | yellow | Features supported by two or more protocols, but not yet bridged between them. | attached tags |
| `compat` | yellow | Protocol differences that need special handling. | AT => AP: escape HTML tags |
| `bug` | red | User-facing breakage and reliability issues within Bridgy Fed. | race conditions |

Additionally, I'd like to add the following two labels for communication and priority:

| name | colour | description | example
|-|-|-|-|
| `see other` | grey | The issue was a duplicate or was fully merged into another issue. | issues newly closed for these reasons |
| `safety` | orange | Related to user safety. | blocking remote users via DM |

@snarfed Does this look good to you? If yes I'd go through the open issues starting with the oldest.
It would likely mess with the 'Most recently updated' view, but I don't think there's really any way around that.

**Comment by Tamschi** [comment]
I'm not entirely sure about `known`. That "standard" name would be `duplicate`, but it always feels a bit disheartening when that's applied to an issue that did provide at least _some_ new information.

**Comment by snarfed** [comment]
This sounds great! Thank you so much, I'm all for it.

I don't feel strongly about known vs duplicate, I'm ok with either. "See other" is another option. Up to you.

**Reply by Tamschi** [reply]
Alright, I'll take care of it.
I forgot that label names here can contain spaces 😅 &ndash; "see other" sound great, I'll edit it into the table so I don't forget.

(I'm not entirely sure when I'll be able to do this. Possibly this week, but more likely early in November.)

**Reply by Tamschi** [reply]
I think I'll quickly add the `safety` label where appropriate though, that should take little time at all.

**Reply by Tamschi** [reply]
I went ahead and created the labels for now (and applied [`safety`](https://github.com/snarfed/bridgy-fed/labels/safety) to the five relevant open ones. I didn't apply it to #1248 since that errs safe now).

Some slight wording changes to the descriptions (length limit, clarification), which I've edited into the table above.

**Reply by Tamschi** [reply]
That should be all of them 😓
It was a bit more work than expected, but hopefully it makes a difference.

**Reply by snarfed** [reply]
It made a HUGE difference. Thank you so much!!!
